.env

```diff
@@ -86,7 +90,7 @@ SITE_LOGO_PATH="build/images/itk-jira-symbol.svg"
 # For Gmail as a transport, use: "gmail://username:password@localhost"
 # For a generic SMTP server, use: "smtp://localhost:25?encryption=&auth_mode="
 # Delivery is disabled by default via "null://localhost"
-MAILER_NAME="ITK"
-MAILER_URL=smtp://mailhog:1025
-MAILER_EMAIL="info@support.srvitkphp72stg.itkdev.dk"
+MAILER_NAME=""
+MAILER_URL=
+MAILER_EMAIL=""
```

Comment:
Set this to some example email address, e.g. `info@jira_economics.docker.localhost`.

Reply:
changed back